standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnHandler.java

```diff
@@ -6396,52 +6430,45 @@ private AbortCompactionResponseElement getAbortCompactionResponseElement(long co
   }
 
   @RetrySemantics.SafeToRetry
-  public AbortCompactionResponseElement abortCompaction(CompactionInfo compactionInfo) throws MetaException {
```

Comment:
any idea why this method is not in CompactioinTxnHandler

Reply:
compact() is also in TxnHandler, and this method aborts the compaction created by the other one. However I'm also not strictly tied to the idea that it must be in TxnHandler.